wpilibc/athena/src/Encoder.cpp

```diff
@@ -132,8 +105,9 @@ Encoder::Encoder(DigitalSource* aSource, DigitalSource* bSource,
       m_bSource(bSource, NullDeleter<DigitalSource>()) {
   if (m_aSource == nullptr || m_bSource == nullptr)
     wpi_setWPIError(NullParameter);
-  else
+  else {
     InitEncoder(reverseDirection, encodingType);
+  }
```

Comment:
Adding braces isn't necessary and is inconsistent with the if portion (also in the next function)